.azure-devops/tox.yml

```diff
@@ -2,109 +2,107 @@ trigger: none
 pr: none
 
 parameters:
-- name: python39
-  displayName: 'Test Python 3.9'
-  type: boolean
-  default: true
-- name: python310
-  displayName: 'Test Python 3.10'
-  type: boolean
-  default: true
-- name: python311
-  displayName: 'Test Python 3.11'
-  type: boolean
-  default: true
-- name: python311
-  displayName: 'Test Python 3.12'
-  type: boolean
-  default: true
-- name: azdev
-  displayName: 'Test development CLI'
-  type: boolean
-  default: true
-- name: azmin
-  displayName: 'Test minimum supported CLI'
-  type: boolean
-  default: true
-- name: azcur
-  displayName: 'Test currently released CLI'
-  type: boolean
-  default: true
+  - name: python39
+    displayName: "Test Python 3.9"
+    type: boolean
+    default: true
+  - name: python310
+    displayName: "Test Python 3.10"
+    type: boolean
+    default: true
+  - name: python311
+    displayName: "Test Python 3.11"
+    type: boolean
+    default: true
+  - name: python311
```

Comment:
python312 lol

Reply:
oh good catch. I don't know if this pipeline runs anymore (github actions checks a lot of our 3.12 stuff) but I'm sure this would have caused an issue somewhere...